File: versioncontrol/src/main/java/ir/ayantech/versioncontrol/model/VCResponseModel.java
package ir.ayantech.versioncontrol.model;

import com.google.gson.Gson;

/**
 * Created by Administrator on 6/10/2017.
 */

public class VCResponseModel {
    private VCStatusModel Status;

    public VCResponseModel(VCStatusModel status) {
        this.Status = status;
    }

    public VCStatusModel getStatus() {
        return Status;
    }

    public void setStatus(VCStatusModel status) {
        this.Status = status;
    }

    public String toString() {
        return new Gson().toJson(this);
    }
}

File: versioncontrol/src/main/java/ir/ayantech/versioncontrol/model/VCResponseModel.kt
package ir.ayantech.versioncontrol.model

import com.google.gson.Gson
import com.google.gson.annotations.SerializedName

open class VCResponseModel(
    @SerializedName("Status")
    var status: VCStatusModel? = null
) {
    override fun toString(): String {
        return Gson().toJson(this)
    }
}
